vhdbuilder/packer/init-variables.sh

```diff
@@ -1,12 +1,38 @@
 #!/bin/bash -e
 set -x
 CDIR=$(dirname "${BASH_SOURCE}")
-
 SETTINGS_JSON="${SETTINGS_JSON:-./packer/settings.json}"
+PUBLISHER_BASE_IMAGE_VERSION_JSON="${PUBLISHER_BASE_IMAGE_VERSION_JSON:-./vhdbuilder/publisher_base_image_version.json}"
+VHD_BUILD_TIMESTAMP_JSON="${VHD_BUILD_TIMESTAMP_JSON:-./vhdbuilder/vhd_build_timestamp.json}"
 SUBSCRIPTION_ID="${SUBSCRIPTION_ID:-$(az account show -o json --query="id" | tr -d '"')}"
 CREATE_TIME="$(date +%s)"
 STORAGE_ACCOUNT_NAME="aksimages${CREATE_TIME}$RANDOM"
 
+# This variable will only be set if a VHD build is triggered from an official branch
+VHD_BUILD_TIMESTAMP=""
+
+# Check if the file exists, if it does, the build is triggered from an official branch
+if [ -f "${PUBLISHER_BASE_IMAGE_VERSION_JSON}" ]; then
+  # Ensure that the file is not empty, this will never happen since automation generates the file after each build but still have this check in place
+  if [ -s "${PUBLISHER_BASE_IMAGE_VERSION_JSON}" ]; then
```

Comment:
should we maybe log something if the file is unexpectedly empty?

Reply:
uh, it wont ever be since automation takes care of it. even if it is, logs here wont help much, we ll just use latest base image, to debug IF its ever empty, we'd need to look at automation where its generated.

Reply:
fair enough, thought might just be nice to have something to grep for in build logs if we ever wanna confirm why the latest image is being used when we expected something else to be used